docs/docs/guides/deployment/keyfactor/signserver.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Use Keyfactor SignServer for attestation signing
+---
+
+### Summary
+Chainloop CLI can sign attestations using a preconfigured SignServer instance, by providing a key reference during the signing process:
+
+```
+chainloop attestation push --key signserver://mysignserver/PlainSigner --signserver-ca-path mysignserver-chain.pem
+```
+
+### Setting up SignServer 
+If your organization doesn't have it already, you can follow these tutorials for a basic setup of Keyfactor's KPI solutions:
+
+* [Quick Start Guide - Start EJBCA Container with Client Certificate Authenticated Access.](https://docs.keyfactor.com/ejbca/latest/quick-start-guide-start-ejbca-container-with-clien)
+* [Quick Start Guide - Issue Client Authentication Certificate using EJBCA](https://docs.keyfactor.com/ejbca/latest/quick-start-guide-issue-client-authentication-cert)
+* [Quick Start Guide - Start SignServer Container with Client Certificate Authenticated Access.](https://docs.keyfactor.com/signserver/latest/quick-start-guide-start-signserver-container-with-)
+
+Finally, you must have a Crypto and Signing workers configured in SignServer. You can follow [this Cosign tutorial](https://docs.keyfactor.com/signserver/latest/tutorial-signserver-container-signing-with-cosign), since the steps for worker configuration are similar.
+
+Once you have it configured, you can reach SignServer signer worker at, for example, `https://mysignserver/PlainSigner`.
+
+Note that, at this moment, SignServer authentication is not implemented.
+
+### Signing Chainloop attestations with SignServer
```

Comment:
We are missing the verification section aren't we?

Reply:
I've added it.